<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Anya Greenberg</title>
    <link rel="stylesheet" href="cv.css">
</head>
<body>

<div class="header">
    <h1 class="name">Anya Greenberg</h1>
    <p class="email">anyadg4@gmail.com</p>
</div>


<h2>EDUCATION</h1>
<hr>

<div class="section">
    <div class="location">
        <span class="company">Harvard T.H. Chan School of Public Health</span>
        <span>Boston, MA</span>
    </div>
    <div class="title_dates">
        <span>M.S. COMPUTATIONAL BIOLOGY AND QUANTITATIVE GENETICS</span>
        <span>Expected May 2026</span>
    </div>
    <br>
</div>

<div class="section">
    <div class="location">
        <span class="company">University of Rochester</span>
        <span>Rochester, NY</span>
    </div>
    <div class="title_dates">
        <span>B.S. DATA SCIENCE, Japanese minor</span>
        <span>May 2020</span>
    </div>
    <ul>
        <li>Organizations: Women in Computing, Data Science Undergraduate Council, Circle K International</li>
    </ul>
    <br>
</div>

<h2>PUBLICATIONS</h2>
<hr>

<div class="section">
    <ul>
        <li><b>Greenberg, A.</b>, et al. Accurate cross-platform GWAS analysis via two-stage imputation. Under revision.</li>
        <li>Ettou, S., <b>Greenberg, A.</b>, et al. FOXC2 and WT1 regulate transcriptional reprogramming during the podocyte response to injury. Under revision. (Co-first author).</li>
        <!-- <li>Wongboonsin, J., <b>Greenberg, A.</b>, et al. The prevalence of genetic forms of nephrotic syndrome in unselected adults within a health care system and their histological and clinical correlates. Under review.</li> -->
        <li>Han, S.K., McNulty, M.T., Benway, C.J., ..., <b>Greenberg A.</b>, et al. (2023) Mapping genomic regulation of kidney disease and traits through high-resolution and interpretable eQTLs. <i>Nature Communications</i>, 14, 2229</li>
        <li>Negm, S., <b>Greenberg, A.</b>, Larracuente, A.M., and Sproul, J.S. (2021) RepeatProfiler: A pipeline for visualization and comparative analysis of repetitive DNA profiles. <i>Molecular Ecology Resources</i>, 21:969-981.</li>
    </ul>
    <br>
</div>

<h2>POSTERS AND PRESENTATIONS</h2>
<hr>

<div class="section">
    <ul>
        <li>"A method to improve GWAS by combining cohorts across genotyping platforms," presented at American Society of Human Genetics Annual Conference. Washington, D.C., November 2023. Poster.</li>
        <li>"FOXC2 and WT1 regulate transcriptional reprogramming during the podocyte response to injury," presented at Nephrology Basic Science and Transplant Research Program Seminar. Boston Children's Hospital, MA. November 2023. Presentation.</li>
        <li>"A two-stage imputation method to combine cohorts genotyped separately," presented at Dr. M. Judah Folkman Research Day. Boston Children's Hospital, MA. June 2023. Poster.</li>
        <li>"Two-stages of imputation for improving GWAS results," presented at Nephrology Basic Science and Transplant Research Program Seminar. Boston Children's Hospital, MA. February 2023. Presentation.</li>
    </ul>
    <br>
</div>

<h2>RESEARCH EXPERIENCE</h2>
<hr>

<div class="section">
    <div class="location">
        <span class="company">Boston Children's Hospital, Nephrology Department</span>
        <span>Boston, MA</span>
    </div>
    <div class="title_dates">
        <span>COMPUTATIONAL RESEARCH ASSISTANT - Lee Lab, Sampson Lab for Kidney Genomics</span>
        <span>Nov 2020 - present</span>
    </div>

    <ul>
        <li>Contributed to multiple research projects to understand the genomic basis of kidney disease (see below)</li>
        <li>Mentored lab members in computational skills such as R and python programming, bash scripting, Anaconda/mamba and virtual environments, SLURM, and various biogrids programs</li>
    </ul>

    <h3>Genome-wide association studies (GWAS) and genotype imputation</h3>
    <ul>
        <li>Developed and validated a new method to improve GWAS using two stages of imputation to facilitate the combination of cohorts genotyped on separate platforms</li>
        <li>Wrote and developed python-based command line tool to facilitate method implementation</li>
        <li>Managed and organized data from multiple sources including UK Biobank, 1000 Genomes Project, and several dbGaP cohorts</li>
        <li>Wrote entire manuscript, forthcoming</li>
    </ul>

    <h3>Differential gene expression and differential binding</h3>
    <ul>
        <li>Performed differential gene and binding analysis of bulk RNA-seq of FACS-sorted podocytes and ChIP-seq of WT1 and FOXC2 to understand the roles of WT1 and FOXC2 in the podocyte response to injury</li>
        <li>Analyzed motif enrichment in WT1 and FOXC2 ChIP-seq peaks to discern potential interactions with other transcription factors</li>
        <li>Wrote results and methods sections for computational analyses in submitted manuscript</li>
    </ul>

    <h3>Allele-specific expression analysis</h3>
    <ul>
        <li>Conducted preliminary allele-specific expression analysis on RNA-seq data from nephrotic syndrome patients using phASER (phasing and allele-specific expression from RNA-seq)</li>
        <li>Assisted clinical research fellow with troubleshooting of phASER, bcftools, samtools, and bash scripting</li>
        <li>Wrote and proofread multiple grants and poster presentations regarding project</li>
    </ul>

    <h3>Variant annotation</h3>
    <ul>
        <li>Wrote python scripts implementing Varsome API, which implements semi-automatic variant annotation based on precomputed metrics</li>
        <li>Provided clinical research fellow with technical support concerning python, bcftools, and bash scripting</li>
        <li>Learned about ACMG classification and the different metrics that determine how variants are classified</li>
    </ul>

    <h3>Expression quantitative trait loci (eQTL) analysis</h3>
    <ul>
        <li>Combined results from bulk RNA-seq, bulk ATAC-seq, and TORUS eQTL analysis to create master table used in publicly available web resource (<a href="https://hugeampkpn.org/research.html?pageid=nephqtl2_home_89">NephQTL2</a>)</li>
        <li>Participated in weekly meetings to discuss the progress and direction of project</li>
        <li>Contributed to writing and proofreading published manuscript</li>
    </ul>

    <h3>Identification of common regulatory elements using machine learning and sequence-based models</h3>
    <ul>
        <li>Performed bulk ATAC-seq peak calling using MACS2 algorithm and gkm-QC for multiple datasets</li>
        <li>Ran gkmSVM and deltaSVM on multiple datasets to identify common regulatory elements and generate testing and training sets for machine learning models</li>
        <li>Genotyped ENCODE data to generate validation dataset for project developing ensemble model of common regulatory element prediction</li>
    </ul>
    <br>
</div>

<div class="section">
    <div class="location">
        <span class="company">STEM-Away Inc., Bioinformatics Pathway</span>
        <span>Remote</span>
    </div>
    <div class="title_dates">
        <span>INTERN, PROJECT LEAD</span>
        <span>Jun 2020 - Aug 2020</span>
    </div>
    <ul>
        <li>Performed quality control, batch correction, differential gene expression analysis, and gene ontology enrichment on two cohorts of gene expression microarray data for colorectal cancer</li>
        <li>Assisted other interns with R programming and debugging</li>
    </ul>
    <br>
</div>

<div class="section">
    <div class="location">
        <span class="company">University of Rochester, Biology Department</span>
        <span>Rochester, NY</span>
    </div>
    <div class="title_dates">
        <span>COMPUTATIONAL RESEARCH ASSISTANT - Larracuente Lab</span>
        <span>Jun 2019 - Nov 2020</span>
    </div>
    <ul>
        <li>Co-developed RepeatProfiler, a program facilitating the analysis of repetitive DNA profiles in the context of evolutionary genomics and speciation of non-model organisms</li>
        <li>Wrote R scritps producing primary output figures of RepeatProfiler</li>
        <li>Built RepeatProfiler docker container for program use in a consistent environment</li>
        <li>Contributed to writing and proofreading published manuscript</li>
    </ul>
    <br>
</div>

<div class="section">
    <div class="title_dates">
        <span>INDEPENDENT STUDY - Larracuente Lab</span>
        <span>Jan 2020 - May 2020</span>
    </div>
    <ul>
        <li>Invesitgated whether distribution of copy number variants can predict recombination rate using supervised and unsupervised machine learning</li>
        <li>Studied the Stephan model of recombination and the evolution of satellite DNA</li>
    </ul>
    <br>
</div>

<div class="section">
    <div class="location">
        <span class="company">University of Rochester Medical Center, Cancer Control</span>
        <span>Rochester, NY</span>
    </div>
    <div class="title_dates">
        <span>DATA SCIENCE CAPSTONE PROJECT</span>
        <span>Jan 2020 - May 2020</span>
    </div>
    <ul>
        <li>Investigated whether patient-reported outcomes concerning adverse events (PRO-CTCAE) could provide potential prognostic insights using statistical analysis and machine learning</li>
        <li>Presented research to diverse audiences including clinical researchers, biostatisticians, data managers, and other health care professionals</li>
    </ul>
    <br>
</div>

<div class="section">
    <div class="title_dates">
        <span>TECHNICAL ASSISTANT</span>
        <span>Jan 2018 - May 2019</span>
    </div>
    <ul>
        <li>Performed quality checks and audits of patient data for multiple studies conducted by the geriatric-oncology team</li>    
        <li>Corresponded professionally with multiple project coordinators to communicate research goals and expectations</li>
    </ul>
    <br>
</div>

<h2>TEACHING/LEADERSHIP EXPERIENCE</h2>
<hr>

<div class="section">
    <div class="location">
        <span class="company">STEM-Away Inc., Bioinformatics Pathway</span>
        <span>Remote</span>
    </div>
    <div class="title_dates">
        <span>BIOINFORMATICS PATHWAY MENTOR</span>
        <span>Oct 2020 - Present</span>
    </div>
    <ul>
        <li>Developed and refined project-based programs for the STEM-Away Inc. virtual internship platform</li>
        <li>Taught webinars on R programming, differential gene expression analysis, and soft skills such as presenting, resume building, and networking</li>
        <li>Provided technical support and feedback to the STEM-Away Inc. network consisting of hundreds of interns</li>
    </ul>
    <br>
</div>

<div class="section">
    <div class="location">
        <span class="company">University of Rochester, Computer Science Department</span>
        <span>Rochester, NY</span>
    </div>
    <div class="title_dates">
        <span>TEACHING ASSISTANT - Intro. to CS, Data Structures and Algorithms</span>
        <span>Jan 2019 - Dec 2019</span>
    </div>
    <ul>
        <li>Led cohorts of students through computer science, data structures, and algorithms workshop problems</li>
        <li>Evaluated student assessments, including quizzes and exams, and provided detailed feedback</li>
        <li>Collaborated with professors and other TAs to identify effective methods of teaching, learning, adn mentorship</li>
    </ul>
    <br>
</div>

<h2>TECHNICAL SKILLS</h2>
<hr>

<div class="section">
    <ul>
        <li>Script writing and programming in R, Python, and Java</li>
        <li>Genomics software including PLINK, bcftools, samtools, and GATK</li>
        <li>Anaconda/mamba and job schedulers (SLRUM, LSF)</li>
    </ul>
    <br>
</div>

<!-- <div class="section">
    <div class="location">
        <span class="company">Company</span>
        <span>Location</span>
    </div>
    <div class="title_dates">
        <span>Title</span>
        <span>Dates</span>
    </div>
    <ul>
        <li>Detail</li>
    </ul>
    <br>
</div> -->

</body>
</html>
